spec/features/guest_spec.rb

```diff
@@ -101,14 +101,28 @@
       end
 
       it 'can remove items from cart' do
-        click_link 'Remove',
-                   href: "/remove_from_cart/#{EquipmentModel.first.id}"
+        @quantity = 0
+
+        fill_in 'quantity_field', with: @quantity.to_s
+        find(:xpath, "//input[@id='quantity_field']").set @quantity.to_s
```

Comment:
Not sure there's a preference, but implementing my `id` suggestion above would make this much simpler: `fill_in "#quantity_field_#{EquipmentModel.first.id}", with: quantity.to_s`